### PR TITLE
Delete Tenant Handle

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: restapi/tenants.go [hunk 1]
```diff
@@ -63,6 +63,30 @@ func registerTenantHandlers(api *operations.M3API) {
...
+}
+
+func getDeleteTenantResponse(params admin_api.DeleteTenantParams) error {
+	opClient, err := operatorClientset.NewForConfig(cluster.GetK8sConfig())
+	if err != nil {
+		return err
+	}
+	currentNamespace := cluster.GetNs()
+
+	err = opClient.OperatorV1().MinIOInstances(currentNamespace).Delete(context.Background(), params.Name, metav1.DeleteOptions{})
```
Comment (Alevsk) [inline restapi/tenants.go]: can you implement this function call inside the `restapi/operator_client.go` ? so we can use it in future testing.

**Review by Alevsk — CHANGES_REQUESTED** [review]

**Review by cesnietor — CHANGES_REQUESTED** [review]
please add tests

**Review by Alevsk — DISMISSED** [review]

File: restapi/tenants.go [hunk 2]
```diff
@@ -63,6 +63,38 @@ func registerTenantHandlers(api *operations.M3API) {
...
+		}
+		return admin_api.NewTenantInfoOK()
+
+	})
+}
+
+// getDeleteTenantResponse gets the output of deleting a minio instance
+func getDeleteTenantResponse(params admin_api.DeleteTenantParams) error {
+	opClientClientSet, err := operatorClientset.NewForConfig(cluster.GetK8sConfig())
+	if err != nil {
```
Comment (cesnietor) [inline restapi/tenants.go:83]: can you add logs for the errors here and on the call of `deleteTenantAction()`

**Review by cesnietor — CHANGES_REQUESTED** [review]

**Review by cesnietor — APPROVED** [review]

File: restapi/tenants.go (hunk 2 above)
Comment (dvaldivia) [reply restapi/tenants.go:83]: we only need to log once the same error, done

**Pull request merged**